srcpkgs/direwolf/patches/20-Fix-musl.patch

```diff
@@ -0,0 +1,37 @@
+Reason: Fixes local declarations when using musl
+
+diff -ru conf/CMakeLists.txt conf/CMakeLists.txt
+--- conf/CMakeLists.txt	2020-10-28 01:45:30.000000000 +0100
++++ conf/CMakeLists.txt	2020-12-29 09:49:13.435238905 +0100
+@@ -26,7 +26,7 @@
+ 
+ # install udev rules for CM108
+ if(LINUX)
+-  install(FILES "${CUSTOM_CONF_DIR}/99-direwolf-cmedia.rules" DESTINATION /etc/udev/rules.d/)
++  install(FILES "${CUSTOM_CONF_DIR}/99-direwolf-cmedia.rules" DESTINATION /usr/lib/udev/rules.d/)
+ endif()
+ 
```

Comment:
Please include this in a separate patch, otherwise it can be removed accidentally if upstream reports that musl compat has been fixed.